src/main/java/commands/ByeCommand.java

```diff
@@ -0,0 +1,38 @@
+package commands;
+
+import java.util.ArrayList;
+
+import storage.Storage;
+import tasks.TaskList;
+import ui.Ui;
+
+/**
+ * The ByeCommand Class inherits Command and is
+ * a specific type of executable command.
+ */
+public final class ByeCommand extends Command {
+
+
+    /**
+     * Constructs the ByeCommand object.
+     *
+     * @param s the entire line of user input
+     */
+    public ByeCommand(ArrayList<String> s) {
+        super(s);
```

Comment:
Should variables with single-character names be used for nested loops only?

Reply:
changed to "userInput", thanks for pointing it out!